translations-app/handleAppTranslations.sh

```diff
@@ -65,7 +65,7 @@ do
   cd translationfiles/templates/
   for file in $(ls)
   do
-    mv $file ../../stable-templates/$version.$RESOURCE_ID.pot
+    mv $file ../../stable-templates/${version/\//-}.$RESOURCE_ID.pot
```

Comment:
All syncs failed tonight with:
```
/handleTranslations.sh: 68: Bad substitution
```

But it works locally:
```sh
$ echo $version
stable23
$ echo ${version/\//-}
stable23
$ echo $?
0
```

Reply:
No idea. 
I tested it also locally with sh.
Maybe do it in a separate call with sed:
replacedVersion=$( echo $version | sed s'#\/#-#'g )

Reply:
Yeag going with that now: `$(echo $version | sed -E 's/\//-/')`
New PR coming